tutorials/2d/custom_drawing_in_2d.rst

```diff
@@ -69,12 +69,14 @@ redrawn if modified:
 
     extends Node2D
 
-    export (Texture) var texture setget _set_texture
+    @export var texture : Texture :
+        set(value):
+            texture = value
+            _set_texture(value)
```

Comment:
Since `_set_texture` is not called anywhere else, I think it makes sense to move its body (`queue_redraw()`) here.